### PR TITLE
Feature/viewbinding sdk 34 migration

### PR DESCRIPTION
**Pull request by fatihergin**

### CODE REVIEW

**Comment by tibbi** [comment]
End call button doesnt have the red bg on neither the accept/decline screen, nor the actual ongoing call screen

**Comment by fatihergin** [comment]
waiting for https://github.com/SimpleMobileTools/Simple-Commons/pull/1802

**Comment by fatihergin** [comment]
end call missing background issue is resolved with [rename ic_call_decline that conflicts with android.core internals](https://github.com/SimpleMobileTools/Simple-Dialer/pull/673/commits/2922e1caad93b17bdb2dbe81f9b4d4ae33ce8b61)

**Comment by FunkyMuse** [comment]
just putting it out here, make sure to also migrate the gradle scripts to `.kts`

File: gradle.properties [hunk 1]
```diff
@@ -17,5 +17,6 @@ org.gradle.jvmargs=-Xmx1536m
 android.useAndroidX=true
 # Automatically convert third-party libraries to use AndroidX
 android.enableJetifier=true
+android.nonTransitiveRClass=false
```
Comment (FunkyMuse) [inline gradle.properties:20]: enable this

**Comment by fatihergin** [comment]
> just putting it out here, make sure to also migrate the gradle scripts to `.kts`

Do we have good reasons to do kts migration in viewbinding task?

File: gradle.properties (hunk 1 above)
Comment (fatihergin) [reply gradle.properties:20]: Why? Just for couple of ms build speed on average? 
Detail your review with reasons please

Comment (FunkyMuse) [reply gradle.properties:20]: 1. They're enabled in all other repos, consistency
2. They're enabled by default anyway
3. Build speed gains
4. Future proofing the project

**Comment by FunkyMuse** [comment]
> > just putting it out here, make sure to also migrate the gradle scripts to `.kts`
> 
> Do we have good reasons to do kts migration in viewbinding task?

Could be another PR

**Comment by tibbi** [comment]
should be good, thanks

**Pull request merged**